custom/icds_reports/urls.py

```diff
@@ -190,7 +191,8 @@
     url(r'^governance_apis/state_names/', GovernanceStateListAPI.as_view(), name='governance_apis_state_names'),
     url(r'^governance_apis/cbe/', GovernanceCBEAPI.as_view(), name='governance_apis_cbe'),
     url(r'^governance_apis/vhsnd/', GovernanceVHNDSAPI.as_view(), name='governance_apis_vhnds'),
-    url(r'^bihar_demographics/household_members_data/', BiharDemographicsAPI.as_view(), name='household_members_data')
+    url(r'^bihar_demographics/household_members_data/', BiharDemographicsAPI.as_view(), name='household_members_data'),
+    url(r'^bihar_demographics/children_and_vaccines/', BiharVaccinesAPI.as_view(), name='children_and_vaccines')
```

Comment:
can you also add you this URL to the here: https://github.com/dimagi/commcare-hq/pull/27183/files#diff-18df09ca9c31f1627e7e56f93f86bc28R15
so that it does it can work when hit by curl.

Reply:
done

Reply:
hey Can't find this added there?